src/main/java/racingcar/controller/Game.java

```diff
@@ -1,75 +1,44 @@
 package racingcar.controller;
 
-import java.util.ArrayList;
 import java.util.List;
 import racingcar.model.Car;
+import racingcar.model.Cars;
+import racingcar.model.Winners;
 import racingcar.view.InputView;
 import racingcar.util.Message;
 import racingcar.util.RandomGenerator;
 import racingcar.view.OutputView;
 
 public class Game {
-    private List<Car> carList = new ArrayList<>();
+
+    private List<Car> cars;
 
     public void play() {
-        InputView inputView = new InputView();
 
-        System.out.println(Message.INPUT_GUIDE_MESSAGE);
-        String[] carNames = inputView.getCarName();
+        OutputView.printMessage(Message.INPUT_GUIDE_MESSAGE);
+        String[] carNames = InputView.getCarName();
 
-        for (String carName:carNames) {
-            Car carInformation = new Car(carName);
-            carList.add(carInformation);
-        }
+        cars = new Cars(carNames).getRacingCars();
```

Comment:
```suggestion
        List<Car> cars = new Cars(carNames);
```
이렇게 해보면 어떨까요?

`moveForwardByCount`는 `car`를 파라미터로 받아줘도 괜챃을 것 같습니다.

Reply:
안녕하세요. 리뷰어님! 
carName을 입력받아 car객체를 생성하는 부분을 Cars로 옮겼습니다.

또한 moveForwardByCount역시 Cars로 옮기는 것이 적합하다 판단되어 옮기고 코드를 수정했습니다.
조언주신대로 car를 파라미터로 받는다면 어떻게 수정할지 고민하다가 위처럼 수정했습니다ㅠㅠ